packages/flutter/lib/src/material/tabs.dart

```diff
@@ -1336,7 +1347,7 @@ class _TabBarViewState extends State<TabBarView> {
     _warpUnderwayCount += 1;
     if (notification is ScrollUpdateNotification && !_controller.indexIsChanging) {
       if ((_pageController.page - _controller.index).abs() > 1.0) {
-        _controller.index = _pageController.page.floor();
+        _controller.index = _pageController.page.round();
```

Comment:
This changed was needed when we allowed user drag interruption during the tab controller animation. Since we now don't allow that to happen, we can revert it back to ```floor()```. However in a similar logic 5 lines below this we use ```round()```, and changing ```floor()``` to ```round()``` doesn't seem to have any side effect.

Reply:
line 1344 definitely has to be round() because we want to achieve the snapping effect. I think this is ok.